### ISSUE TITLE
Has the author stopped maintaining this project?

### ISSUE THREAD

**Issue by Alvin-shao** [jr-robotics/robo-gym#78]
Why hasn't anyone replied to the questions in the past year? I need help! @ @matteolucchi @friedemannzindler @breigo

**Comment by matteolucchi** [comment]
Dear @Alvin-shao, 

Thank you for the interest in this project. I am personally not working on this project anymore and this is currently on low maintenance mode. There have been replies in the past 2 weeks and not over a year ago like you mentioned. 

I encourage you to look through the previous issues and comments to find additional insights to solve your problems. Other users might find time to help you with your issues but please keep in mind that this is not their duty.

Moreover, I must emphasize that tagging me and other people multiple times in the same issue will not increase the chances of getting a reply.  So I kindly ask you to stop doing that to keep the repository and the issue section clean and easier to maintain. 

I wish you all the best with solving your issue. 

Best, 

Matteo

**Comment by Alvin-shao** [comment]
> 

Thank you for your reply. I have been trying my best to find your contact information these days, but they have all failed. It's really regrettable to hear that you are no longer working on this project. I think your project is great. The problem I encountered was just a seemingly small one, but the entire project cannot be used, so I feel very painful. If the problem I submitted earlier caused you any trouble, I apologize.

**Comment by Alvin-shao** [comment]
> 亲爱的，
> 
> 感谢您对这个项目的关注。我个人不再从事这个项目了，目前处于低维护模式。在过去的两周里有回复，而不是像你提到的一年多前。
> 
> 我鼓励您浏览之前的问题和评论，以找到解决您的问题的其他见解。其他用户可能会抽出时间帮助您解决问题，但请记住，这不是他们的职责。
> 
> 此外，我必须强调，在同一问题中多次标记我和其他人不会增加获得回复的机会。因此，我恳请您停止这样做，以保持存储库和问题部分的清洁和更易于维护。
> 
> 祝您一切顺利地解决您的问题。
> 
> 最好
> 
> 马迪奥

Thank you for your reply. I have been trying my best to find your contact information these days, but they have all failed. It's really regrettable to hear that you are no longer working on this project. I think your project is great. The problem I encountered was just a seemingly small one, but the entire project cannot be used, so I feel very painful. If the problem I submitted earlier caused you any trouble, I apologize.
